docs/source/using-discovery/bash.rst

```diff
@@ -0,0 +1,867 @@
+
+.. note:: 
+  Similar to remove, ``cp`` works for files; to copy a folder, along with its contents, it must be done recursively via ``cp -r <FOLDER_PATH> <DESTINATION>``.
+
+``mv`` - Move or rename a file or directory.
+
+.. code-block:: shell
+  :emphasize-lines: 3
+
+  mv file2.txt file4.txt
+  ls
+  file3.txt file4.txt directory1 directory2
+
+``cat`` - Concatenate and display the contents of one or more files.
+
+.. code-block:: shell
+  :emphasize-lines: 2
+
+  cat file3.txt
+  This is the contents of file3.txt
+
+``grep`` - Search for a pattern in a file or input.
+
+.. code-block:: shell
+  :emphasize-lines: 2
+
+  grep "the" file3.txt
+  This is the contents of file3.txt
+
+``sort`` - Sort the lines of a file or input.
+
+.. code-block:: shell
+  :emphasize-lines: 2
+
+  sort file3.txt
+  This is the contents of file3.txt
+
+``uniq`` - Remove duplicates from a sorted file or input.
+
+.. code-block:: shell
+  :emphasize-lines: 2
+
+  sort file3.txt | uniq
+  This is the contents of file3.txt
+
+``wc`` - Count the number of lines, words, and characters in a file or input.
+
+.. code-block:: shell
+  :emphasize-lines: 2
+
+   wc file3.txt
+   1   4  26 file3.txt
+
+``head`` - Display the first lines of a file or input.
+
+.. code-block:: shell
+  :emphasize-lines: 2
+
+   head file3.txt
+   This is the contents of file3.txt
+
+``tail`` - Display the last lines of a file or input.
+
+.. code-block:: shell
+  :emphasize-lines: 2
+
+  tail file3.txt
+  This is the contents of file3.txt
+
+``less`` - View the contents of a file one page at a time.
+
+.. code-block:: shell
+
+   less file3.txt
+
+``top`` - Show the currently running processes and system information.
+
+.. code-block:: shell
+
+   top
+
+To exit, press ``q``.
+
+``ps`` - Show information about the currently running processes.
+
+.. code-block:: shell
+
+   ps
+.. note::
+   ``<pid>`` (``PID``) in the command should be replaced with the actual process ID of the process you want to terminate; the output of the ``kill`` command will typically be empty unless there is an error in executing the command.
+
+It's essential to be cautious when using the ``kill`` command, as terminating a process can cause data loss or corruption. Therefore, before using ``kill``, you should always try to gracefully stop the process by sending a termination signal, such as ``SIGTERM``, first. If that does not work, you can try a stronger signal, such as ``SIGKILL``.
+
+``kill`` - Terminate a process by its process ID::
+
+   kill <pid>
+
+Advanced Commands
+=================
+In this section we will provide examples of some helpful advanced commands, and then take a closer look at three essential advanced commands.
+
+``sed`` - Stream editor for filtering and transforming text.
+
+.. code-block:: shell
+  :emphasize-lines: 2-4, 7-9
+
+  cat file1.txt
+  This is line 1
+  This is line 2
+  This is line 3
+
+  sed 's/line 1/Line 1/' file1.txt
+  This is Line 1
```

Comment:
This change was a little subtle. I wonder if you can change the word entirely? sub "line" for "cat" or something more noticible?